.circleci/config.yml

```diff
@@ -2,6 +2,7 @@
 # PACKET_API_TOKEN
 # PACKET_PROJECT_ID
 # PACKET_SSH_PUB_KEY
+# VAGRANT_CLOUD_TOKEN
```

Comment:
@warrenvw Were there any other ENV variables in previous PRs that we didn't include in the list?

Reply:
NVM, found what was missing and added it

Reply:
👍